ddtrace/settings/__init__.py

```diff
@@ -5,9 +5,6 @@
 from .integration import IntegrationConfig
 
 
-# Default global config
-config = Config()
```

Comment:
This should be moved to `ddtrace/__init__.py`

Reply:
Ended up not being possible as there's a circular dependency between `ddtrace` and `ddtrace.monkey`